packages/react-scripts/backpack-addons/splitChunks.test.js

```diff
@@ -0,0 +1,234 @@
+'use strict';
+
+jest.mock('../config/paths', () => ({
+  appPackageJson: './test/mockPackage.json',
+}));
+
+describe('splitChunks', () => {
+  const mockData = {
+    name: 'test',
+    version: '1.0.0',
+    'backpack-react-scripts': {},
+  };
+
+  let isEnvDevelopment = true;
+
+  beforeEach(() => {
+    jest.resetModules();
+  });
+
+  test('should return default if no config defined', () => {
+    jest.doMock('./test/mockPackage.json', () => ({
```

Comment:
The testing setup here was a bit awkward, open to suggestions as always.
Essentially, `splitChunks.js` pulls the JSON here:

```
const paths = require('../config/paths');
const appPackageJson = require(paths.appPackageJson);
const bpkReactScriptsConfig = appPackageJson['backpack-react-scripts'] || {};
```

So in the test file, we're mocking the result of `paths` to be `{appPackageJson: './test/mockPackage.json'}`.
The exact path is arbitrary, only matters that we stay consistent.

Then the second line will resolve as `const appPackageJson = require('./test/mockPackage.json');`.

So then in every test we can mock out that module using `doMock`. We need `doMock` instead of `.mock` such that the calls aren't hoisted and overriden. 

We can't (or at least I can't) use the standard jest module pattern because the module returns a JSON object, not a function.

https://jestjs.io/docs/jest-object#jestdomockmodulename-factory-options